lambdas/indexer/test/README.md

```diff
@@ -1,9 +1,10 @@
 # Rudimentary tests
 
 ## TODO
+
 * [ ] run inside of Docker container that mimics Lambda env
-		* https://blog.quiltdata.com/an-easier-way-to-build-lambda-deployment-packages-with-docker-instead-of-ec2-9050cd486ba8
-		* https://nvbn.github.io/2015/09/09/pytest-docker/
+  *<https://blog.quiltdata.com/an-easier-way-to-build-lambda-deployment-packages-with-docker-instead-of-ec2-9050cd486ba8>
+  * <https://nvbn.github.io/2015/09/09/pytest-docker/>
```

Comment:
Huh?

Reply:
Should there be the nested list with links?

![image](https://github.com/user-attachments/assets/d15b1e75-3a87-425d-b151-1ae320cc2ac0)

Reply:
fixed, thanks!